proto/cosmos/accounts/vesting/v1/module.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package cosmos.accounts.vesting.v1;
+
+import "cosmos/app/v1alpha1/module.proto";
+
+// Module is the config object of the vesting module.
```

Comment:
Why is this needed? This isn't a module in the same sense as existing modules

Reply:
will remove